### ISSUE TITLE
Allow MicroExpress to listen on a socket file instead of a port.

### ISSUE THREAD

**Issue by jdrouet** [NozeIO/MicroExpress#4]
I'd like to be able to do

```swift
let app = Express()

app.listen("/my/awesome/service.socket")
```

Like with the original express (due to node [http module](https://nodejs.org/api/net.html#net_server_listen))

If this looks doable and interesting, I could take a look at it.

**Comment by helje5** [comment]
That would be really cool! You would probably just have to add a related bootstrap to SwiftNIO first.

**Comment by helje5** [comment]
Implemented in PR #5, thanks!
